docs/content/preview/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -525,3 +525,22 @@ amount        | 1000
 txn_type      | credit
 created_at    | 2020-11-07 22:09:04.8537
 ```
+
+## Step 7. Fault tolerance during a region outage
```

Comment:
Why not just call this Leader preference and set the preferred leader to any zone within the same region?

Reply:
The latency between different zones of the same region is less than 1ms. Within the same zone latency can be anywhere from 0.2ms to almost 1ms depending on which rack and fault domains they run on.
Only in cross region where latency is tens of milliseconds we see a significant improvement.